How do you install this?

I know this is probably a stupid question to those who know what they are doing....

How do I actually install and use this library. I've downloaded it and ran CMAKE but it gives me an error about not knowing where Qt is. When I give it the directory it still complains about a missing cmake file. Is there an example of this file or should it already be in one of the sub directories. 

Should the Qt Directory be the root of Qt or the root of the specific version that I want to use (inside the Qt dir)?

Sorry for the dumb issue but we all start somewhere... :)

Hi, I am trying to build this on debian linux (9.2/stretch).
After `apt-get install qtbase5-dev` I did like you proposed:
```
mkdir build && cd build
export CMAKE_PREFIX_PATH=/usr/include/x86_64-linux-gnu/qt5
cmake .. && make
```

I get the following error:

> CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Qt5Core/Qt5CoreMacros.cmake:63 (file):
>   file attempted to create a directory:
>   /home/user/workspace/nodeeditor/resources into a source directory.
> Call Stack (most recent call first):
>   /usr/lib/x86_64-linux-gnu/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (qt5_make_output_file)
>   /usr/lib/x86_64-linux-gnu/cmake/Qt5Core/Qt5CoreMacros.cmake:282 (_QT5_PARSE_QRC_FILE)
>   CMakeLists.txt:49 (qt5_add_resources)
> 
> 
> -- Configuring incomplete, errors occurred!
> See also "/home/user/workspace/nodeeditor/CMakeFiles/CMakeOutput.log".

Do I need more than qtbase5-dev? Do I point it to the right directory?

--- a/src/NodeGraphicsObject.cpp
+++ b/src/NodeGraphicsObject.cpp
@@ -177,8 +177,9 @@ QVariant NodeGraphicsObject::itemChange(GraphicsItemChange change, const QVarian
 
 void NodeGraphicsObject::mousePressEvent(QGraphicsSceneMouseEvent *event)
 {
-    //if (_nodeState.locked())
-    //return;
+    if (graphModel().nodeFlags(_nodeId) & NodeFlag::Locked) {
+        return;
+    }
 
     AbstractNodeGeometry &geometry = nodeScene()->nodeGeometry();
 

